packages/shared/lib/models/Activity.ts

```diff
@@ -37,6 +38,7 @@ export enum LogActionEnum {
     SYNC = 'sync',
     SYNC_CLIENT = 'sync client',
     SYNC_DEPLOY = 'sync deploy',
+    SYNC_FREQUENCY = 'sync frequency update',
```

Comment:
Is this going to appear in the Activity tab? I'm not sure it should. IMO it's just some management action (like you would do on the webapp) + you can check if it worked by looking at the connection's sync schedule.

Reply:
I don't have a full picture on how customers are using this tab, I had the impression it was more or less an audit log, so I would expect to find this action in there too.

Reply:
The activity log entry is a bit weird as well.

1. It is unresolved (not successful or failed) 
2. The type doesn't show up

![image](https://github.com/NangoHQ/nango/assets/1724137/5f5c5e03-5086-4325-9bc6-bfaaec242624)

In short I think we can keep this out of the activity tab as Bastien mentions

Reply:
Alright 👌🏻